.github/pull_request_template.md

```diff
@@ -1,29 +1,29 @@
 ## Proposed changes
 
-Describe the big picture of your changes here to communicate to the maintainers why the pull request is necessary and why we should accept it. If it fixes a previously reported bug, be sure to link to that issue. Please also summarize the changes to the code.
+Your contribution to PorePy is greatly appreciated. Give the maintainers a clear explanation of why your pull request (PR) is needed and why we should accept it. If this PR is solving an issue, explain how it is done. Please, also summarise the changes to the code. 
```

Comment:
See my comment below on the use of personal pronomina. My opinion is the same her.

Reply:
I have adapted the text. Please, see if this is close to your suggestion.